Header          header
uint32          week
float64         timeOfWeek
uint32          insStatus
uint32          hdwStatus
float32[3]      theta
float32[3]      uvw
float64[3]      lla
float32[3]      ned
